client/src/views/K8s.vue

```diff
@@ -156,10 +156,11 @@
                       icon="fa-solid fa-eye"
                     />
                   </template>
-                  <v-card width="50%" class="mx-auto pa-5">
-                    <v-icon class="ml-auto" @click="dialog = false"
-                      >mdi-close</v-icon
-                    >
+                  <v-card
```

Comment:
why close button removed?

Reply:
we have many dialogs for workers it fires all together it closes when you click outside, will figure it out later https://github.com/codescalers/cloud4students/issues/238